src/views/Home/Home.tsx

```diff
@@ -45,10 +46,26 @@ function Home(props: Props): React.ReactElement {
   const [inviteModal, setInviteModal] = useState(false);
   const window = useWindowDimensions();
   const safeArea = useSafeAreaInsets();
+  const [msg, setMsg] = useState("");
+  const [offline, setOffline] = useState(false)
 
   const fetchCultures = async () => {
-    let cultureNames = await Culture.list();
-    setCultures(cultureNames);
+    try {
+      const cultures = await Culture.list();
+      setCultures(cultures);
+    } catch (err) {
+      if (err instanceof OfflineError) {
+        const ledger = await Ledger.list();
+        let cultures = [];
+        ledger.forEach(
+          (val, key) => cultures.push({ name: key, modified: cultures[key] })
+        );
+        setCultures(cultures)
+        setOffline(true)
```

Comment:
Shoot, we forgot this!

`Ledger.list` can throw an error so try catch it.

```ts
      try {
          const ledger = await Ledger.list();
          let cultures = [];
          ledger.forEach(
            (val, key) => cultures.push({ name: key, modified: cultures[key] })
          );
          setCultures(cultures)
          setOffline(true)
       } catch (err) {
          setMsg(err.toString());
       }
```

Not a suggestion, because there's no way I can format the code correctly on web.

Reply:
Just fixed it.